doc/topics/development/contributing.rst

```diff
@@ -209,8 +247,8 @@ Fork a Repo Guide_>`_ and is well worth reading.
 
     Pull request against `saltstack/salt`_ are automatically tested on a
     variety of operating systems and configurations. On average these tests
-    take 30 minutes.  Depending on your GitHub notification settings you may
-    also receive an email message about the test results.
+    take a couple of hours.  Depending on your GitHub notification settings
+    you may also receive an email message about the test results.
```

Comment:
We send out emails through GH? 
I don't think we do, do we?

Reply:
GH might send emails - we don't.

Reply:
(they [can] send a bunch of different notifications depending on what your settings are)

Reply:
(they [can] send a bunch of different notifications depending on what your settings are)